packages/_usa-password/src/index.mjs

```diff
@@ -0,0 +1,17 @@
+import behavior from "../../uswds-core/src/js/utils/behavior.mjs";
+import toggleFormInput from "../../uswds-core/src/js/utils/toggle-form-input.mjs";
+import { CLICK } from "../../uswds-core/src/js/events.mjs";
+import { prefix as PREFIX } from "../../uswds-core/src/js/config.mjs";
```

Comment:
File extension for relative files is a requirement for mocha. We could turn this off, but I wanted to see how it affected package exports.

For now, it seems like there is **no** effect.